bench/slhdsa.cc

```
@@ -0,0 +1,72 @@
+
+#include <benchmark/benchmark.h>
+
+#include <openssl/base.h>
+#include <openssl/err.h>
+#include <openssl/evp.h>
+#include <openssl/slhdsa.h>
+
+#include "./internal.h"
+
+namespace {
+void BM_SpeedSLHDSA(benchmark::State &state) {
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)